src/components/ScriptureCard.js

```diff
@@ -1,56 +1,65 @@
-import { useEffect } from 'react'
+import React from 'react';
 import PropTypes from 'prop-types'
-import {
-  Card,
-  CardContent,
-  useContent,
-  useCardState,
-} from 'translation-helps-rcl'
-import { ScripturePane, useScripture } from 'single-scripture-rcl'
+import { Card, useCardState } from 'translation-helps-rcl'
+import { ScripturePane } from 'single-scripture-rcl'
 import { getLanguage } from '@common/languages'
+import { ComboBox } from '@components/ComboBox';
+import { getItemByTitle, updateTitle } from '@utils/ScriptureVersionHistory';
+import { getScriptureVersionSettings, useScriptureSettings } from '@hooks/useScriptureSettings';
 
-export default function ScriptureCard({
-  title,
-  chapter,
-  verse,
-  server,
-  owner,
-  branch,
-  languageId,
-  classes,
-  bookId,
-  resourceId,
-  disableWordPopover,
-}) {
-  const scriptureConfig = useScripture({
-    reference: {
-      projectId: bookId,
-      chapter,
-      verse,
-    },
-    resource: {
-      languageId,
-      projectId: resourceId,
-      owner,
-      branch,
-    },
-    config: {
-      server,
-      cache: { maxAge: 1 * 1 * 1 * 60 * 1000 },
-    },
-    disableWordPopover,
-  })
+const label = 'Version';
+const style = { marginTop: '16px', width: '500px' };
+
+export default function ScriptureCard(Props) {
+  const {
```

Comment:
Are these values ( title, classes) even used anywhere?

Reply:
yes, passed down to Card (see line 76)